docs/plugins/config-yml-changes.md

```diff
@@ -0,0 +1,49 @@
+# Add new config to the {root}/config/opensearch-dashboards.yml file
+
+1. Add new config value in the opensearch_dashboards.yml file. Below is an example where I'm adding new flag to hide/show
+`wizard` plugin.
+
+```yml
+
+# Set the value of this setting to true to start exploring wizard
+# functionality in Visualization.
+# wizard.enabled: true
+```
+
+Note that the yml config values are read as path to the plugin config schema file. In the above example, `wizard` is the `Id` of the plugin to which I'm introducing this new config value.
```

Comment:
```suggestion
Note that the yml config values are read as path to the plugin config schema file. In the above example, `wizard` is the `Id` of the plugin to which I'm introducing this new config value. The value should match the default, so that if a user uncomments the line, it will still have the default behavior.
```